Isosurfacing_3/include/CGAL/Isosurfacing_3/internal/Cell_type.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2022 INRIA Sophia-Antipolis (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Julian Stahl
+
+#ifndef CGAL_DOMAIN_CELL_TYPE
+#define CGAL_DOMAIN_CELL_TYPE
+
+#include <CGAL/license/Isosurfacing_3.h>
+
+#include <limits>
+
+namespace CGAL {
+namespace Isosurfacing {
+
+
+// Was supposed to check if an algorithm can handle a specific domain. Not used right now.
```

Comment:
this file seems to be deletable (or it should use a class enum instead and fix the "POLYHERDAL" typo)